frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts

```diff
@@ -82,16 +85,21 @@ export const selectWorkerRows = async () => {
     splitedRow = newCurrRow.split(/\s/g);
     if (splitedRow[3] === 'worker') {
       workerAz.push(splitedRow[4]);
-      await table
-        .$$('input')
-        .get(index)
-        .click();
-      selectedNodes.push(
-        await table
-          .$$('a')
+      if (
+        !(await $$('tbody input')
```

Comment:
Can you use `class` or `id` if its there? Selector `tbody input` is not very reliable.

Reply:
it's the best that there is. for the situation.

Reply:
`tbody input` has not other clashing instance in the page.